web/themes/custom/hatter/templates/page/page.html.twig

```diff
@@ -0,0 +1,85 @@
+{#
+/**
+* @file
+* Theme override to display a single page.
+*
+* The doctype, html, head and body tags are not in this template. Instead they
+* can be found in the html.html.twig template in this directory.
+*
+* Available variables:
+*
+* General utility variables:
+* - base_path: The base URL path of the Drupal installation. Will usually be
+*   "/" unless you have installed Drupal in a sub-directory.
+* - is_front: A flag indicating if the current page is the front page.
+* - logged_in: A flag indicating if the user is registered and signed in.
+* - is_admin: A flag indicating if the user has permission to access
+*   administration pages.
+*
+* Site identity:
+* - front_page: The URL of the front page. Use this instead of base_path when
+*   linking to the front page. This includes the language domain or prefix.
+*
+* Page content (in order of occurrence in the default page.html.twig):
+* - messages: Status and error messages. Should be displayed prominently.
+* - node: Fully loaded node, if there is an automatically-loaded node
+*   associated with the page and the node ID is the second argument in the
+*   page's path (e.g. node/12345 and node/12345/revisions, but not
+*   comment/reply/12345).
+*
+* Regions:
+* - page.header: Items for the header region.
+* - page.primary_menu: Items for the primary menu region.
+* - page.secondary_menu: Items for the secondary menu region.
+* - page.highlighted: Items for the highlighted content region.
+* - page.help: Dynamic help text, mostly for admin pages.
+* - page.content: The main content of the current page.
+* - page.sidebar_first: Items for the first sidebar.
+* - page.sidebar_second: Items for the second sidebar.
+* - page.footer: Items for the footer region.
+* - page.breadcrumb: Items for the breadcrumb region.
+*
+* @see template_preprocess_page()
+* @see html.html.twig
+*
+* @copyright Copyright (c) 2017 Palantir.net
+*/
+#}
+
+{{ page.branding }}
+
+{% if page.header|render|trim is not empty  %}
+  <header class="site-header">
+    <div class="container">{{ page.header }}</div>
+  </header>
+{% endif %}
+
+{% if page.content|render|trim is not empty %}
+  <section class="banner">
+    <div class="container">{{ page.content }}</div>
+  </section>
+{% endif %}
+
+{% if page.featured_bottom_first|render|trim is not empty %}
+  <section class="map-banner">
+    {{ page.featured_bottom_first }}
+  </section>
+{% endif %}
+
+{% if page.featured_bottom_second|render|trim is not empty %}
+  <section class="banner">
+    {{ page.featured_bottom_second }}
+  </section>
+{% endif %}
+
+{% if page.footer_left or page.footer_right %}
+  <footer class="site-footer">
+    <div class="container">
+      <div class="l-2up--1">{{ page.footer_left }}</div>
+      <div class="l-2up--2">{{ page.footer_right }}</div>
+      <div class="l-1up">
```

Comment:
And this.

Reply:
This is there to specifically match the prototype - https://midcamp.github.io/Hatter/home/

Reply:
https://github.com/MidCamp/Hatter/pull/7 :)

Reply:
@froboy that doesn't help with the footer nav vs. footer block, which is what this does